silverback/cluster/client.py

```diff
@@ -296,6 +296,14 @@ def bots(self) -> dict[str, Bot]:
         handle_error_with_response(response)
         return {bot.name: bot for bot in map(Bot.model_validate, response.json())}
 
+    def bots_list(self) -> dict[str, list[Bot]]:
+        response = self.get("/bots")
+        handle_error_with_response(response)
+        bots_dict: dict[str, list[Bot]] = {}
```

Comment:
In general, using `collections.defaultdict` may be a better solution to this type of issue where you want a default value for a key in the dict